isis/tests/FunctionalTestsSpiceinit.cpp

```diff
@@ -528,21 +528,21 @@ TEST(Spiceinit, TestSpiceinitPadding) {
 TEST_F(DefaultCube, TestSpiceinitCsmCleanup) {
   // Add stuff from csminit
   testCube->putGroup(PvlGroup("CsmInfo"));
-  StringBlob testBlob("test string", "CSMState");
+  Blob testBlob("CSMState", "String");
```

Comment:
Something I mentioned to Austin was that the blob type should probably be an enum to properly define a selection of blob types. Right now it just exists as a string so the type could be anything.

Reply:
We could do something like:

* Binary <- Blob default
* String
* Table
* OriginalLabel
* OriginalXmlLabel
* History
* Stretch
* Polygon

Reply:
I think this looks good! I would rather see Geometry over Polygon but either works

Reply:
These are the currently used types except for Binary. We need to maintain backwards compatibility.

Reply:
This is going to potentially cause some weird merge conflicts with other stuff, so I'm going to put this is in as a separate PR after this.